plugin-script-node/src/test/java/io/kestra/plugin/scripts/node/CommandTest.java

```diff
@@ -0,0 +1,100 @@
+package io.kestra.plugin.scripts.node;
+
+import com.google.common.collect.ImmutableMap;
+import io.kestra.core.models.executions.AbstractMetricEntry;
+import io.kestra.core.models.executions.LogEntry;
+import io.kestra.core.queues.QueueFactoryInterface;
+import io.kestra.core.queues.QueueInterface;
+import io.kestra.core.runners.RunContext;
+import io.kestra.core.runners.RunContextFactory;
+import io.kestra.core.storages.StorageInterface;
+import io.kestra.core.utils.TestsUtils;
+import io.kestra.plugin.scripts.exec.scripts.models.DockerOptions;
+import io.kestra.plugin.scripts.exec.scripts.models.RunnerType;
+import io.kestra.plugin.scripts.exec.scripts.models.ScriptOutput;
+import io.kestra.plugin.scripts.exec.scripts.runners.BashException;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import jakarta.inject.Inject;
+import jakarta.inject.Named;
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@MicronautTest
+class CommandTest {
+    @Inject
+    RunContextFactory runContextFactory;
+
+    @Inject
+    StorageInterface storageInterface;
+
+    @Inject
+    @Named(QueueFactoryInterface.WORKERTASKLOG_NAMED)
+    private QueueInterface<LogEntry> logQueue;
+
+    @Test
+    void task() throws Exception {
+        List<LogEntry> logs = new ArrayList<>();
+        logQueue.receive(logs::add);
+
+        URI put = storageInterface.put(
+            new URI("/file/storage/get.yml"),
+            IOUtils.toInputStream(
+                "console.log('yeah baby!');",
```

Comment:
```suggestion
                "console.log('hey there!');",
```
this way seems a bit more... polite? 🤣